python/ray/air/integrations/wandb.py

```diff
@@ -541,6 +542,7 @@ def __init__(
         log_config: bool = False,
         upload_checkpoints: bool = False,
         save_checkpoints: bool = False,
+        upload_timeout: int = DEFAULT_SYNC_TIMEOUT,
```

Comment:
This is 30 mins right now. I want am thinking about shortening the max time, and also logging a warning if this final upload takes longer than ~1 minute.

Reply:
@krfricke

Reply:
I'm happy either way. 30 minutes does seem a bit much, but since we wait for that long in regular syncs as well, how about we keep it and see if we encounter it at all?